getAlby/lndhub.go
# Feature/accept keysend

Closes #169 

Process incoming keysend payments. The user login should be present in a TLV with type `696969`, as specified [here](https://github.com/satoshisstream/satoshis.stream/blob/main/TLV_registry.md#field-696969---lnpay), and should be hex-encoded, although the decoding is done automatically by LND.

TLV records are stored in the database as json blobs with the invoices, and are returned by the `/getuserinvoices` endpoint (the values are base64 encoded):

```
...
        "amt": 100,
        "custom_records": {
            "696969": "TE9JMXdwQVdidnh4dE40aHB3Wno="
        },
        "description": "Keysend payment",
....
```

To test:
- Add an `lndhub.regtest.getalby.com` wallet to your Alby extension.
- Add simnet-lnd-2 to your Alby extension ([Instructions](https://github.com/getAlby/lightning-browser-extension/wiki/Test-setup#lnd-2))
- Use the following pen to send a keysend payment: https://codepen.io/kiwiidb/pen/MWQpjyp . Fill in your `lndhub.regtest.getalby.com` user ID in the last input field. Load the `simnet-lnd-2` wallet in the Alby extension and make the payment. The  payment should arrive in the `lndhub.regtest.getalby.com` account.
